### PR TITLE
React-Login-Demo

### PR DESCRIPTION
**Pull request by aditya822024**
### What do you want to achieve?
- Every time a new user logs in with their new email ID, the new chat widget should open, logging out the old chat widget.

### PR Checklist

- [ ] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Review by prochnost04 — CHANGES_REQUESTED** [review]
This solution will confuse the customer

File: example/React-Demo/src/chat.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect } from "react";
+import React from "react";
```
Comment (panwaranuj01) [inline example/React-Demo/src/chat.js:2]: add one import statement

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]

File: example/React-Demo/src/chat.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    if (window.kommunicate) {
+      return;
+    }
+    (function (d, m) {
+      var kommunicateSettings = {
+        appId: "kommunicate-support",
+        popupWidget: true,
+        automaticChatOpenOnNavigation: true,
+      };
+      if ((mail ?? "").length > 0) {
```
Comment (panwaranuj01) [inline example/React-Demo/src/chat.js:15]: we can simplify this

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]
First file it for the `development` branch @aditya822024

File: example/React-Demo/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "login-page",
+  "version": "0.1.0",
+  "private": true,
```
Comment (panwaranuj01) [inline example/React-Demo/package.json:4]: Is it required?

File: example/React-Demo/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  "dependencies": {
+    "@kommunicate/kommunicate-chatbot-plugin": "0.0.7",
```
Comment (panwaranuj01) [inline example/React-Demo/package.json:6]: If you are not using this we can remove this as well

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]

**Comment by panwaranuj01** [comment]
@aditya822024 If it's not needed then please close this pr.

**Comment by devashishmamgain** [comment]
It is needed as a demo link to share with customers in future if anyone
asks for it.
we can remove the unnecessary files from it, and keep only the relevant
ones, along with a readme.md file that explains the concept

**Pull request closed without merging**